skunkworks_crow/src/main/java/org/odk/share/views/ui/bluetooth/BtSenderActivity.java

```diff
@@ -133,6 +136,16 @@ private Disposable addUploadEventSubscription() {
                 }, Timber::e);
     }
 
+    /**
+     * Enable the bluetooth discovery for other devices. The timeout is 60 seconds.
+     */
+    private void enableDiscovery() {
```

Comment:
And I think sender should be made `DISCOVERABLE` not the receiver so that it would be able to see the sender in its scanned results.

Reply:
This method allows sender to be `DISCOVERABLE` for a specific duration, by default, it is 120s. And I set a fixed 60s for that.  I think this change can help receiver to show all the possible senders and help them establish a valid connection in a fixed time, since in the next step we are going  to build a default method, we need that timeout I think. 

As for who should scan bluetooth devices, I think sender should host a file server so it should be `DISCOVERABLE`, the connect invitation should be sent by the receiver.

Reply:
I don't think just by hosting the file the sender is made discoverable.  Please check that on multiple devices.

Reply:
Bluetooth doesn't support multiple devices, the files server should be ready first, so the sender should get ready before the receiver. The connection order should start with the sender then the receiver, so letting receiver to connect a ready server is no issues.

Reply:
By multiple devices, I mean to say that try to test this behavior in different devices, not by running them all at once.